geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreFactory.scala

```diff
@@ -185,7 +187,8 @@ object AccumuloDataStoreParams {
   val recordThreadsParam   = new Param("recordThreads", classOf[Integer], "The number of threads to use for record retrieval", false, 10)
   val writeThreadsParam    = new Param("writeThreads", classOf[Integer], "The number of threads to use for writing records", false, 10)
   val looseBBoxParam       = new Param("looseBoundingBox", classOf[java.lang.Boolean], "Use loose bounding boxes - queries will be faster but may return extraneous results", false, true)
-  val statsParam           = new Param("collectStats", classOf[java.lang.Boolean], "Toggle collection of statistics", false, true)
+  val generateStatsParam   = new Param("generateStats", classOf[java.lang.Boolean], "Toggle collection of data statistics", false, true)
+  val queryAuditingParam   = new Param("auditQueries", classOf[java.lang.Boolean], "Toggle collection of query statistics", false, true)
```

Comment:
looking forward wondering if "auditQueries" might conflict with actual query auditing. maybe "collectQueryStats" ? Unless maybe we think it might go through the same mechanism